lib/suspenders/app_builder.rb

```diff
@@ -145,6 +145,10 @@ def use_postgres_config_template
         force: true
     end
 
+    def install_mongoid
+      run 'rails generate mongoid:config'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.